pkg/systemd/host.js

```diff
@@ -1410,7 +1410,8 @@ PageCpuStatus.prototype = {
 
         machine_info().
             done(function (info) {
-                self.plot.set_yaxis_max(info.cpus * 1000);
+                n_cpus = info.cpus;
+                self.plot.set_yaxis_max(n_cpus * 1000);
```

Comment:
I don't quite understand the introduction of the `n_cpus` variable here. Changing it here won't magically update `options.yaxis` and the graphs, will it? You have to call `self.plot.set_yaxis_max` to set the new value. So what's the effect of this and the previous hunk?

Reply:
> (Hmm, it would be better to explicitly refresh the plot...)

`set_yaxis_max` already does this.

Reply:
> I'll add a comment.

Done.